### ISSUE TITLE
Foto van poster nieuwsbericht/activiteit?

### ISSUE THREAD

**Issue by Maharball1** [lesteenman/pcsa-drupal-theme#96]
Voor de ouwe lullen is het wellicht handig om naast een naam een visuele weergave van die persoon te hebben. Wellicht een foto van de poster naast/bij het nieuwsbericht/activiteit? Ben benieuwd wat @PauluzzS @lesteenman @jpingen @MangoTree009  er van vinden. Vooral voor Heeren die elkaar nauwelijks zien is het wellicht makkelijk om een connectie te leggen dan.

**Comment by jpingen** [comment]
Mergen met #38  ?

**Comment by lesteenman** [comment]
Als dit iets is dat we bij beide willen, en het in principe de enige grote change nog is, zou ik #38 sluiten en dit als de issue laten staan. Eens? Of zijn er meer aanmerkingen bij de activiteiten pagina die niet een losse issue waard zijn?

**Comment by PauluzzS** [comment]
Ik hoef niet zonodig de kop van degene te zien die een activiteit plaatst, of die een nieuwsbericht post.. Ik zie liever iets dat het nieuwsbericht of de activiteit illustreert. Een plaatje van een borrel bij een borrel bijvoorbeeld. Wie het erop zet doet er niet zoveel toe, zou praesidium moeten zijn voor veel dingen. Wellicht als #65 is geimplementeerd dat het wel waardevol wordt omdat ik dan wel graag het logo van het college er bijvoorbeeld bij zou zien.

**Comment by lesteenman** [comment]
Dat was in eerste instantie ook mijn gedachte, maar misschien is het leuk
om er klein bij te hebben (als je ingelogd bent?). Foto's kun je al
plaatsen bij nieuws artikelen, trouwens.

**Comment by PauluzzS** [comment]
Misschien...
Foto's plaatsen bij artikelen is inderdaad fijn.
Wellicht kunnen we voor activietien ook standaardfoto's of veelgebruikte
foto's suggereren? Veel activiteiten zijn behoorlijk repetitief. Maar moet
miss een separate issue van maken..

Groet,

Paul L. Smits

2017-09-13 23:51 GMT+02:00 Erik <notifications@github.com>:

> Dat was in eerste instantie ook mijn gedachte, maar misschien is het leuk
> om er klein bij te hebben (als je ingelogd bent?). Foto's kun je al
> plaatsen bij nieuws artikelen, trouwens.
>

**Comment by lesteenman** [comment]
Ja, is denk ik los.

Nu ik er nog wat over nagedacht heb is denk ik een foto van de poster niet
echt relevant, omdat je post vanuit het dispuut, niet jezelf. Comments
daarentegen zijn echt vanuit jezelf.

**Comment by Maharball1** [comment]
Activiteiten en nieuwsberichten kunnen ook prima van mezelf zijn. Zie
bijvoorbeeld nieuwsbericht van Pico over hun vakantie.

Op 14 sep. 2017 00:25 schreef "Erik" <notifications@github.com>:

> Ja, is denk ik los.
>
> Nu ik er nog wat over nagedacht heb is denk ik een foto van de poster niet
> echt relevant, omdat je post vanuit het dispuut, niet jezelf. Comments
> daarentegen zijn echt vanuit jezelf.
>

**Comment by lesteenman** [comment]
Fair point! Misschien is dat een eerste stap voor commissie-posts, om aan
te kunnen geven of een post een persoonlijke of algemene post is.

**Comment by PauluzzS** [comment]
zou de woorden 'persoonlijk' en 'algemeen' niet gebruiken omdat ze
verwarrend zijn met de publiek-checkbox. Wellicht een 'namens' selector met
'mijzelf', 'het dispuut', 'college x', 'commissie y'.
Maar dit moet in een andere issue 👮

Groet,

Paul L. Smits

2017-09-14 0:31 GMT+02:00 Erik <notifications@github.com>:

> Fair point! Misschien is dat een eerste stap voor commissie-posts, om aan
> te kunnen geven of een post een persoonlijke of algemene post is.
>
